content/lua/concepts/mathematical-library/terms/ult/ult.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: 'ult()'
+Description: 'Compares the absolute value of two integers and returns boolean true if the first is less than the second.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Math'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+The method **`ult()`** takes two integers as arguments and compares their absolute values. The method will return the boolean value `true` if and only if the first value is less than the second value. Otherwise, it will return `false`.
```

Comment:
```suggestion
The **`ult()`** method takes two integers as arguments and compares their absolute values. The method will return `true` if and only if the first value is less than the second value. Otherwise, it will return `false`.
```